HW_1/task_2.py

```diff
@@ -0,0 +1,12 @@
+# Найти сумму числе трехзначного числа.
+print("Введите трехзначное число", end=" :")
+n = int(input())
+if 99 < n < 1000:
+    count = 0
+    while n > 0:
+        d = n % 10
+        count += d
```

Comment:
Значение переменной d используются только в одном месте. Значит, оно не обязательно к сохранению в переменной.
```suggestion
        count += n % 10
```